packages/turndown-plugin-gfm/src/tables.js

```diff
@@ -98,10 +98,23 @@ rules.table = {
       }
     }
 
-    return '\n\n' + emptyHeader + content + '\n\n'
+    var captionContent = node.caption == null ? '' : node.caption.textContent || '';
+    var caption = captionContent === '' ? '' : captionContent + '\n\n';
```

Comment:
Can we do something like this instead? `const captionContent = node.caption?.textContent || ''`

Also for new code please use `const` (or `let` but not `var`)

Reply:
> Can we do something like this instead?

absolutely - I wasn't sure if I was allowed to use new syntax in this file given how old the syntax around is

---

edit:

I'm having a build error when I try to use optional chaining:

```yaml
❯ yarn build-all

> @joplin/turndown-plugin-gfm@1.0.54 build-cjs
> rollup -c config/rollup.config.cjs.js && rollup -c config/rollup.config.browser.cjs.js


src/gfm.js → lib/turndown-plugin-gfm.cjs.js...
[!] Error: Unexpected token
src/tables.js (101:40)
 99:     }
100:
101:     const captionContent = node.caption?.textContent || '';
                                             ^
102:     const caption = captionContent === '' ? '' : `${captionContent}\n\n`;
103:     const tableContent = `${emptyHeader}${content}`.trimStart();

npm ERR! Lifecycle script `build-cjs` failed with error:
npm ERR! Error: command failed
npm ERR!   in workspace: @joplin/turndown-plugin-gfm@1.0.54
npm ERR!   at location: /Users/maxpatiiuk/site/git/joplin/packages/turndown-plugin-gfm
```

Looks like rollup version is 6 years old:

```yaml
❯ yarn rollup --version
rollup version 0.50.1
```

6 years worth of updates probably would have some breaking changes, so updating rollup is likely outside the scope of this PR.

Though, I was able to update the code to use `const` and template literals